src/routes/users.js

```diff
@@ -0,0 +1,116 @@
+import { list, readOne, deleteOne, patchOne, createOne } from 'mongoose-crudl'
+
+import allowAccessTo from 'bearer-jwt-auth'
+import AccountModel from '../models/Account.js'
+import UserModel from '../models/User.js'
+import { MethodNotAllowedError, ValidationError } from 'standard-api-errors'
+import jwt from 'jsonwebtoken'
+import crypto from 'crypto'
+
+export default (apiServer) => {
+  const secrets = process.env.SECRETS.split(' ')
+
+  apiServer.patch('/v1/accounts/:accountId/users/:id/name', async req => {
+    allowAccessTo(req, secrets, [{ type: 'admin' }, { type: 'user', role: 'admin' }, { type: 'user', user: { _id: req.params.id } }])
+    const user = await patchOne(UserModel, { id: req.params.id, accountId: req.params.accountId }, { name: req.body.name })
+    return user
+  })
+
+  apiServer.patch('/v1/accounts/:accountId/users/:id/password', async req => {
+    allowAccessTo(req, secrets, [{ type: 'admin' }, { type: 'user', role: 'admin' }, { type: 'user', user: { _id: req.params.id } }])
```

Comment:
here as well. I know, that the user id can't be the same for two users, but still